src/app/contexts/RequestContext/index.jsx

```diff
@@ -41,7 +41,7 @@ export const RequestContextProvider = ({
   const env = getEnv(origin);
   const platform = isAmp ? 'amp' : 'canonical';
   const statsDestination = getStatsDestination({
-    isUK,
+    isUk: platform === 'amp' ? true : isUK,
```

Comment:
Thanks for this change @HarryVerhoef 🙂 Perhaps we should add a comment as you suggested here to explain this?

Also, is there anyway we could add a test for this? We could assert that outside the uk on amp the statsdestination defaults to PS?

Reply:
Thanks @andrewscfc!
Yeah good ideas, I'll do both now 👍